capellambse_context_diagrams/collectors/class_tree.py

```diff
@@ -0,0 +1,88 @@
+# SPDX-FileCopyrightText: 2022 Copyright DB Netz AG and the capellambse-context-diagrams contributors
+# SPDX-License-Identifier: Apache-2.0
+"""This submodule defines the collector for the Class-Tree diagram."""
+from __future__ import annotations
+
+import collections.abc as cabc
+import typing as t
+
+from capellambse import helpers
+from capellambse.model.crosslayer import information
+
+from .. import _elkjs, context
+from . import generic, makers
+
+LAYOUT_OPTIONS: _elkjs.LayoutOptions = {
+    "algorithm": "layered",
+    "edgeRouting": "ORTHOGONAL",
+    "elk.direction": "DOWN",
+    "partitioning.activate": True,
```

Comment:
What's up with this global `"partitioning.activate": True` here? Isn't this supposed to be configurable? Does this even have any effect?

Reply:
This needs to be set else partitioning isn't enabled at all. ELK seems to need this global switch.